src/modules/auto_leave_raid/index.js

```diff
@@ -0,0 +1,55 @@
+const $ = require('jquery');
+const twitch = require('../../utils/twitch');
+const debug = require('../../utils/debug');
+const settings = require('../../settings');
+const watcher = require('../../watcher');
+const domObserver = require('../../observers/dom');
+
+const COMMUNITY_HIGHLIGHT_SELECTOR = '.community-highlight-stack__card';
+
+let autoLeaveRaidListener;
+
+function handleLeaveRaid(node, isConnected) {
+    if (!isConnected) return;
+    const $node = $(node);
+    const raidBanner = $node.find('[data-test-selector="raid-banner"]');
+    if (raidBanner.length > 0) {
+        const raidContext = twitch.getChannelRaidContext(raidBanner.get(0));
```

Comment:
do we actually need this raid context? can we detect this from the button state?